tests/ci/build-release.yaml

```diff
@@ -0,0 +1,107 @@
+steps:
+- id: 'create .git/ folder'
+  name: 'alpine/git:v2.24.1'
+  entrypoint: '/bin/sh'
+  args:
+  - '-ec'
+  - |
+    if [[ ! -d ".git/" ]]; then
+      if [[ "$_GITHUB_API_TOKEN" == "na" ]]; then
+        echo "Expecting value for _GITHUB_API_TOKEN"
+        exit 1
+      fi
+      git init
+      git remote add origin https://$_GITHUB_API_TOKEN@github.com/vectorizedio/redpanda
+      echo "Checking out COMMIT=${COMMIT_SHA}"
+      git fetch --depth=200 origin $COMMIT_SHA
+      git reset --hard FETCH_HEAD
+
+      # fetch tags. if no commit found, fetch more.
+      # fetch everything if tag is still not found.
+      git fetch --tags origin
+      if ! git describe; then
+        git fetch --deepen=500 origin
+        if ! git describe; then
+          git fetch --unshallow origin || true
+        fi
+      fi
+    fi
+
+    git clone --depth 1 -b meta_pipable https://$_GITHUB_API_TOKEN@github.com/0xdiba/meta vtools
+    cp vtools/.vtools.yml .
+
+- id: 'install vtools'
+  name: 'gcr.io/redpandaci/fedora:31-python37-docker1903'
+  entrypoint: bash
+  args:
+  - -ec
+  - |
+    # show CPU info
+    cat /proc/cpuinfo | sed -e '/^$/,$d'
+
+    # install vtools
+    python3 -mvenv --system-site-packages build/venv/v
+    source build/venv/v/bin/activate
+    pip install tools/
+
+    vtools install infra-deps --conf=vtools/vtools/artifacts/ci/vtools-gcc-release.yml
```

Comment:
we should make the conf an env var. 

`export CONF=..`
`vtools install infra-deps $CONF`